apps/api/src/app/integrations/usecases/remove-integration/remove-integration.usecase.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Scope } from '@nestjs/common';
+import { IntegrationRepository } from '@notifire/dal';
+import { RemoveIntegrationCommand } from './remove-integration.command';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+
+@Injectable({
+  scope: Scope.REQUEST,
+})
+export class RemoveIntegration {
+  constructor(private integrationRepository: IntegrationRepository) {}
+
+  async execute(command: RemoveIntegrationCommand) {
+    try {
+      await this.integrationRepository.delete({ _id: command.integrationId });
```

Comment:
It's really important to add the _applicationId here, otherwise every user will be able to remove integrations of other users just by knowing their id :)

Reply:
Yea you're right I will add the app id here as well!

Reply:
Also here, not sure I see this change. It's still deleting by using only the integrationId. Could you check this out? In this case anyone can remove any integration he wants by just having the id of it

Reply:
Strange I'm sure I added app id as well I will look into it.